lib/Steps/Integrations/NextJs.ts

```diff
@@ -437,4 +457,58 @@ export class NextJs extends BaseIntegration {
     arr.splice(start, deleteCount, ...inserts);
     return arr;
   }
+
+  private async _mergeNextConfig(
+    destinationPath: string,
+    templatePath: string,
+    destinationDir: string,
+    templateFile: string,
+    configDirectory: string,
+    mergeableFilePath: string,
+  ): Promise<void> {
+    // if no next.config.js exists, we'll create one
+    if (!fs.existsSync(destinationPath)) {
+      fs.copyFileSync(templatePath, destinationPath);
```

Comment:
The written file is gonna be a bit ugly because of the `// ORIGINAL CONFIG` comment. Can we either replace it with something a little more instructive or remove it before writing it to disk?

We should probably replace it even with

```
module.exports = {
  // Your Next.js configuration here
}
```

or smth like that.

Reply:
If no next.config.js exists, then it will use the existing `next.config.js` template instead of the new one with the `// ORIGINAL CONFIG` line.

Reply:
Ah you're right. I hope this isn't just me but to me the entire logic of this file is super hard to get behind. We should def clean this up some time (not your code but the one that existed before - it probably just grew over time).